parm/config/config.fcst

```diff
@@ -161,6 +161,9 @@ export isatmedmf=1
 tbf=""
 if [[ "$satmedmf" = ".true." ]]; then tbf="_satmedmf" ; fi
 
+#Convection schemes
+export progsigma=".true."
```

Comment:
The field table is only being updated for Thompson, but this shows up in the namelist for all MP.

Reply:
@lisa-bengtsson can you reply to this comment.

Reply:
This has now been updated @WalterKolczynski-NOAA  - I'm resolving the comment but please feel free to reopen if there are outstanding issues.

Reply:
Is the only difference with all these just this last `sigmab` field? And it is the same for all? Any reason we can't append a table fragment to the field table rather than doubling the number of field tables?

Reply:
This follows the format for the other field tables.   Could the re-engineering of this (sounds like a good idea) be tabled to a different PR? I believe that we discussed opening an issue for this at a coupled model meeting at one point.  I do know there's an open model issue to generate this on the fly as well.

Reply:
Yes